docs/production-deployment/self-hosted-guide/checklist.mdx

```diff
@@ -46,106 +77,81 @@ At a high level, you will want to track these 3 categories of metrics:
+We recommend testing for this availability level while load testing.
+We also recommend validating this level of reliability while doing server upgrades, to ensure no loss of service availability.  
 
-### Continue-As-New on demand
+Temporal Clusters can be deployed in as many regions as needed to meet various requirements:
+- Data Residency
+- Latency
+- Security / Isolation
+This can multiply the effort to implement and maintain clusters.
 
-Set up a mechanism to force your Workflow to Continue-As-New on demand.
-The easiest way to do this is to define a Signal handler that continues the Workflow when received.
-Be sure that all previously received Signals have been processed and that all running [Child Workflows](/encyclopedia/child-workflows) have been waited on or otherwise dealt with.
+[Temporal Cloud is available in 15 AWS regions](/cloud/service-availability).
 
-The newly started Workflow then unpacks the state in the input and can "resume" the Workflow according to your business logic.
+## Management and Control Plane 
 
-A huge benefit of this approach is that you avoid making patches to running Workflows.
-Instead, tell them all to continue onto a new [Task Queue](/workers#task-queue) with [Workers](/workers) that have the updated Workflow code.
+Temporal success leads to larger Temporal deployments.
+Needs can increase, and can go from having one or two production use cases in a single region to many use cases in many regions.
+Running multiple clusters is complex work, as each needs its own setup, tuning, and configuration. 
 
-### When and how to make changes to your Workflow code
+This operational management pain leads to a need to monitor and manage all of your Temporal Services in a unified way.
+To do this, you need a layer on top of Temporal to manage multiple clusters: a control plane.
+Temporal does not ship an open source control plane. This can be complex to do and take significant effort.
 
-When you have an understanding of the importance of writing deterministic Workflow code, you might find yourself grappling with the best way to make changes to that code.
+With Temporal Cloud, all namespaces in all regions can be managed from a single view.
+The Temporal Cloud control plane allows operators to observe, manage, and control.
+Responsibilities for individual namespaces can be delegated using RBAC built into [Temporal Cloud](https://temporal.io/cloud).
 
-Three approaches are available to you at this time: Task Queue–based versioning, Workflow Name–based versioning, and the Patch and GetVersion APIs.
-Each approach has merits and demerits.
-(We're also [working toward](https://github.com/temporalio/proposals/blob/master/versioning/worker-versions.md) making Task Queue–based versioning a first-class concept built into Temporal, but that's not quite ready.)
+Self-hosted Temporal does not support RBAC or audit logging out of the box.
```

Comment:
I'd flip this sentence and the next one, so the solution first and then the caveat (TCloud provides. Unfortunately, self-hosted doesn't)

Reply:
Keeping this one the way it is after making the previous change, because that way we're still ending the section on a positive note with Cloud.